src/Objectivity.AutoFixture.XUnit2.AutoMoq/SpecimenBuilder/IgnoreVirtualMembersSpecimenBuilder.cs

```diff
@@ -0,0 +1,30 @@
+namespace Objectivity.AutoFixture.XUnit2.AutoMoq.SpecimenBuilder
+{
+    using System;
+    using System.Reflection;
+    using Ploeh.AutoFixture.Kernel;
+
+    internal class IgnoreVirtualMembersSpecimenBuilder : ISpecimenBuilder
+    {
+        public object Create(object request, ISpecimenContext context)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
+
+            var pi = request as PropertyInfo;
+            if (pi == null)
+            {
+                return new NoSpecimen();
+            }
+
+            if (pi.GetGetMethod().IsVirtual)
```

Comment:
In this case I would leave my implementation. It is easier to understand what this code is doing.